### ISSUE TITLE
A11y_SQLAzureDataStudio_ExportBacpac_SummaryScreen_Keyboard:The data in the last column is not visible in potrait mode.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9255]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.4;#Orientation;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.3.4

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select forth radio button.
9. Navigate till next button & hit enter.
10. Provide all the details & hit enter.
11. Change the orientaion to potraite mode.

**Actual:**
When the screen is set to portrait mode the data in the last column of the table is not visible and the arrow keys do not move to the right side. 

**Same issue is observed in the following page:**
1.Import Bacpac-Summary Screen
2.SQL Server on Windows
3. SQL Server Big Data Cluster-Summary

**Expected:**
The data should be clearly visible on the screen, there should be available method to move to the right of the screen with arrow keys.

**User Impact:**
Keyboard users will not be able to view the data present on the screen.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EVeL8F_88R1Er5KsfcdlUy8BB0AbLiwyfJhGN-5aeIcIUg?e=TsrdLa

**Comment by alanrenmsft** [comment]
@fsteffi I am not able to reproduce the issue at least for resource deployment dialog, could you please try the latest insider build and also fix your attachment, i'm not able to open the link

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by Roopesh-T** [comment]
![image](https://user-images.githubusercontent.com/86601127/217482549-e03616c8-792e-408d-93b0-ccc106bdf526.png)
In land scape mode the issue is not repro.
As per the new guidelines closing the bug as Orientation is not testing for PC Machine.

**Git Hub**: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
